client/component-lab/Banner/StoryStyles.stylus

```diff
@@ -0,0 +1,18 @@
+.BusinessAddOnBanner
+  margin-bottom                     30px
+  padding					  		48px
```

Comment:
perhaps switch between tabs and spaces was jumping back and forth after each opening of file